Graph-Templates.md

```diff
@@ -29,19 +29,31 @@ Option | Description
 --- | ---
 *Delete* | Remove the **Graph Template** if it's *Deletable*
 *Duplicate* | Make an exact copy of the **Graph Template**
-*Resize* | Make a resize decision that affects both the **Graph Template** and any **Graphs** created using this **Graph Template**
-*Sync Graphs* | Update all **Graphs** using this **Graph Template** with the latest definition, adding new *Graph Items* and removing orphaned *Graph Items*
+*Resize* | Make a resize decision that affects both the **Graph Template** and any
+ **Graphs** created using this **Graph Template**
+*Sync Graphs* | Update all **Graphs** using this **Graph Template** with the
+latest definition, adding new *Graph Items* and removing orphaned *Graph Items*
 
 When Editing a **Graph Template**, there will be several sections that will
 require information from the Administrator.  Those sections include:
 
 Section | Description
 --- | ---
 *Graph Template Items* | These items paint inside the canvas of the **Graph**
-*Graph Item Inputs* | This list is created automatically when you add **Data Sources** to a **Graph Template**.  However, you can also override certain **Data Template** and **Graph Template** fields by adding specifically named objects to this section as well.
-*Graph Template* | Allows you to name the **Graph Template**, and whether or not you will allow multiple instances of this **Graph Template** to be used for a **Device**, and whether or not to test for valid data before allowing the creation of **Graphs**.  The *Test Data Source* option is useful when you want to leverage a complex **Device Template** that contains **Graph Templates** that may not be applicable for all classes of **Devices** using the **Graph Template**.
-*Common Options* | Things like Title, Vertical Label, Image Format, Height and Width, Base Value and Slope Mode
-*Scaling Options* | Which determine if the Graph is auto-scaled and by what means.  Determines if the Graph will have fixed upper and lower limits
+*Graph Item Inputs* | This list is created automatically when you add **Data Sources**
+ to a **Graph Template**.  However, you can also override certain **Data Template**
+ and **Graph Template** fields by adding specifically named objects to this
+section as well.
+*Graph Template* | Allows you to name the **Graph Template**, and whether or not
+you will allow multiple instances of this **Graph Template** to be used for a
+**Device**, and whether or not to test for valid data before allowing the creation
+of **Graphs**.  The *Test Data Source* option is useful when you want to leverage
+complex **Device Template** that contains **Graph Templates** that may not be applicable
+for all classes of **Devices** using the **Graph Template**.
```

Comment:
This is one kinds of area where sticking to the 80 column limit should be avoided.  Tables are not so easy to read when they are split like this.